### PR TITLE
[Kokoro] Support for TextFields, Math, and Typography

### PR DESCRIPTION
**Pull request by willlarche**

### CODE REVIEW

File: components/TextFields/BUILD [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    sdk_frameworks = [
+        "QuartzCore",
+        "UIKit",
+    ],
+    deps = [
+      "//components/AnimationTiming",
+      "//components/private/Math",
+      "//components/Palettes",
+      "//components/Typography"
+      ],
```
Comment (jverkoey) [inline components/TextFields/BUILD]: Minor nit: `]` indentation should line up with `deps`.

File: components/TextFields/BUILD [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+)
+
+swift_library(
+    name = "unit_test_swift_sources",
+    srcs = glob(["tests/unit/*.swift"]),
+    deps = [
+      ":TextFields",
+      "//components/private/Math",
+      "//components/Palettes",
+      ],
```
Comment (jverkoey) [inline components/TextFields/BUILD]: indentation here as well.

File: components/TextFields/BUILD [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+ios_unit_test(
+    name = "unit_tests",
+    deps = [
+      ":unit_test_sources",
+      ":unit_test_swift_sources",
+    ],
+    minimum_os_version = "8.0",
+    visibility = ["//visibility:private"],
+)
```
Comment (jverkoey) [inline components/TextFields/BUILD]: No newline at the end of the file - not sure if this matters but may be safer to add one.

File: components/Typography/BUILD [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+mdc_public_objc_library(
+    name = "Typography",
+    sdk_frameworks = [
+        "QuartzCore",
+        "UIKit",
+    ],
+    deps = [
+      "//components/private/Application",
+      "@material_internationalization_ios//:MDFInternationalization"
+      ],
```
Comment (jverkoey) [inline components/Typography/BUILD]: Indentation.

File: components/Typography/BUILD [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+)
+
+ios_unit_test(
+    name = "unit_tests",
+    deps = [
+      ":unit_test_sources",
+    ],
+    minimum_os_version = "8.0",
+    visibility = ["//visibility:private"],
+)
```
Comment (jverkoey) [inline components/Typography/BUILD]: End of file newline

**Review by jverkoey — APPROVED** [review]
lgtm after stylistic nits are fixed.

File: components/Typography/BUILD (hunk 5 above)
Comment (willlarche) [reply components/Typography/BUILD]: Done

File: components/Typography/BUILD (hunk 4 above)
Comment (willlarche) [reply components/Typography/BUILD]: Done

File: components/TextFields/BUILD (hunk 3 above)
Comment (willlarche) [reply components/TextFields/BUILD]: Done

File: components/TextFields/BUILD (hunk 2 above)
Comment (willlarche) [reply components/TextFields/BUILD]: Done

File: components/TextFields/BUILD (hunk 1 above)
Comment (willlarche) [reply components/TextFields/BUILD]: Actually, now it's done.

File: components/TextFields/BUILD (hunk 3 above)
Comment (willlarche) [reply components/TextFields/BUILD]: Hmmm. Where did it go?

**Comment by ajsecord** [comment]
Please use buildifier for BUILD file formatting, manual formatting should never be needed.

**Review by ianegordon — APPROVED** [review]

**Comment by ianegordon** [comment]
LGTM, please merge when Travis is happy.

**Pull request merged**